bioinput.py

```diff
@@ -1,10 +1,32 @@
-class Bioinput():
-    """
+import pandas as pd
+
+
+class Bioinput:
+    """Input Class
 
     """
 
     def __init__(self, file="ecg_data.csv"):
         self.file = file
 
     def read_input(self):
-        print("BME590")
\ No newline at end of file
+        """Read in a CSV file and convert it into a DataFrame with the columns being time and voltage"
+
+        :return: An DataFrame for the time and voltage ecg data
+        """
+
+        try:
+            ecg_dataframe = pd.read_csv(self.file, names=['time', 'voltage'], skiprows=1)
+            if len(ecg_dataframe) == 0:
+                print("No data found")
+                return ecg_dataframe
+            ecg_dataframe['time'] = pd.to_numeric(ecg_dataframe['time'], errors='coerce')
+            ecg_dataframe['voltage'] = pd.to_numeric(ecg_dataframe['voltage'], errors='coerce')
+            ecg_dataframe = ecg_dataframe.dropna()
+
+        except FileNotFoundError:
+            raise FileNotFoundError("File Not Found")
+        except ValueError:
+            raise ValueError("Invalid data, Non-Numeric types present")
+        else:
+            return ecg_dataframe
```

Comment:
I'm not sure what is going on here... why is `ecg_dataframe` not an attribute of the `Bioinput` object that is created?

Reply:
So should we make this an attribute and then just return self.ecg_dataframe??

Reply:
No need to return it; once established as an attribute, it will be available in the created object.